themes/default/content/blog/pulumi-cli-time-elapsed/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Pulumi CLI now displays Time Elapsed per resource"
+date: 2022-11-02T11:45:35-06:00
+draft: false
+meta_desc: See deployment times across different types of AWS and GCP resources.
+authors:
+    - kyle-dixler
+tags: ["pulumi", "AWS", "GCP", "features"]
+---
+
+If you’ve deployed resources to your favorite cloud provider, you have probably found yourself sitting in the console thinking: “I don’t know how long this is going to take.” Then you deploy the resource and think: “When did I even start this?” When using Pulumi, the CLI prints out how long the update took after it ran, but while you’re in the moment, it feels like ages.
+
+## We’re excited to announce a CLI usability enhancement.
+
+**You can now see how long each of your resources are taking to deploy.**
+
+When you run `pulumi up`, you’ll see that individual resources have the time taken displayed during updates including up, destroy, import, and refresh.
+
+[![asciicast](https://asciinema.org/a/5Qf4oLoYOWvUHUMUXcPjHvXxL.svg)](https://asciinema.org/a/5Qf4oLoYOWvUHUMUXcPjHvXxL)
```

Comment:
Did you know we also have an `asciicast` shortcode?

```suggestion
{{< asciicast id="5Qf4oLoYOWvUHUMUXcPjHvXxL" >}}
```

Lets you play in place without having to link out!